### PR TITLE
Changes for master that are important

### PR DESCRIPTION
**Pull request by supertopher**
we have added some ruby functionality to our text file repo

### CODE REVIEW

File: bestruby.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+put "this is broken"
```
Comment (dreeve) [inline bestruby.rb]: Hey my dude, you forgot the s in `puts`.

:metal:

**Comment by supertopher** [comment]
doh!

File: bestruby.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+puts "this is broken"
```
Comment (dreeve) [inline bestruby.rb:1]: It's no longer broken...

**Comment by dreeve** [comment]
Syntactically great, but the message is a little confusing. We'll save that for a future release.

**Pull request merged**